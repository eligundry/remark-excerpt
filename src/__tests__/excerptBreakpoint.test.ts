import { excerptBreakpoint, ExcerptBreakpointOptions } from '../index'

describe('excerptBreakpoint', () => {})
