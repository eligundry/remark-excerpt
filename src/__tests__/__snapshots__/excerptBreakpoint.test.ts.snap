// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`excerptBreakpoint should handle identifier with dashes 1`] = `
# Title

Paragraph 1.

Paragraph 2.

<span id="read-more" />

Paragraph 3.

Paragraph 4.

`;

exports[`excerptBreakpoint should handle identifier with spaces 1`] = `
# Title

Paragraph 1.

Paragraph 2.

<span id="read-more" />

Paragraph 3.

Paragraph 4.

`;

exports[`excerptBreakpoint should handle mdx 1`] = `
import something from 'something'

export const meta = {
  prop: 'value',
}

# Title

Paragraph 1.

Paragraph 2.

<span id="read-more" />

Paragraph 3.

Paragraph 4.

`;

exports[`excerptBreakpoint should handle no spacing in exceprt comment 1`] = `
# Title

Paragraph 1.

Paragraph 2.

<span id="read-more" />

Paragraph 3.

Paragraph 4.

`;

exports[`excerptBreakpoint should return after excerpt when multiple exist 1`] = `
# Title

Paragraph 1.

<!-- excerpt -->

Paragraph 2.

<!-- excerpt -->

Paragraph 3.

<span id="read-more" />

Paragraph 4.

`;

exports[`excerptBreakpoint should return excerpt breakpoint using "exceprt" identifier 1`] = `
# Title

Paragraph 1.

Paragraph 2.

<span id="read-more" />

Paragraph 3.

Paragraph 4.

`;

exports[`excerptBreakpoint should return excerpt breakpoint using "more" identifier 1`] = `
# Title

Paragraph 1.

Paragraph 2.

<span id="read-more" />

Paragraph 3.

Paragraph 4.

`;

exports[`excerptBreakpoint should return excerpt using "preview" identifier 1`] = `
# Title

Paragraph 1.

Paragraph 2.

<span id="read-more" />

Paragraph 3.

Paragraph 4.

`;

exports[`excerptBreakpoint should return excerpt using "teaser" identifier 1`] = `
# Title

Paragraph 1.

Paragraph 2.

<span id="read-more" />

Paragraph 3.

Paragraph 4.

`;

exports[`excerptBreakpoint should return excerpt using custom identifier 1`] = `
# Title

Paragraph 1.

Paragraph 2.

<span id="read-more" />

Paragraph 3.

Paragraph 4.

`;

exports[`excerptBreakpoint should return unmodified document when no excerpt exists 1`] = `
# Title

Paragraph 1.

Paragraph 2.

Paragraph 3.

Paragraph 4.

`;
